src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -24,6 +24,9 @@ public Settings(string serviceName = null)
                 ServiceName = DEFAULT_SERVICE_NAME;
             }
 
+            // Overwrite the service name if it is specified in ENVVAR, reg, or config file
+            ServiceName = SettingsReader<string>.Read("InternalQueueName", ServiceName);
+
             {
```

Comment:
Same here if we remove the block { } above let's remove it here, if not, then lets not.